src/redux/books/books.js

```diff
@@ -0,0 +1,45 @@
+// Action Types
+const BOOK_ADDED = 'bookStore/books/BOOK_ADDED';
+const BOOK_REMOVED = 'bookStore/books/BOOK_REMOVED';
+
+/**
+ * @function addBook - the action creator for adding a book
+ * @param {Object} payload - the payload for adding a new Book
+ * @returns {Object} - the action object
+ */
```

Comment:
- [ ] remove commented code

Reply:
Ok, thanks. But it is actually not commented code but JSDoc comment